### ISSUE TITLE
implement trust and flag

### ISSUE THREAD

**Issue by pfraze** [ssbc/phoenix#243]
there are now trust and flag buttons. they should emit messages and have complete flows (ui copy explaining what they mean)

proposing these message schemas

```js
{ type: 'trust', rel: 'trusts', feed: - } // create trust edge
{ type: 'trust', rel: 'untrusts', feed: - } // delete trust edge
{ type: 'flag', rel: 'flags', feed: - } // create flag edge
{ type: 'flag', rel: 'unflags', feed: - } // delete flag edge
```

@dominictarr how do you feel about those?

**Comment by dominictarr** [comment]
looks fine.

**Comment by pfraze** [comment]
closed by #248
